tests/modules/consent/lib/Auth/Process/ConsentTest.php

```diff
@@ -31,6 +31,9 @@ private function processFilter(array $config, array $request)
      */
     public function testCheckDisable()
     {
+        // Consent code path attempts to load metadata, which tries to find a config.php
+        putenv('SIMPLESAMLPHP_CONFIG_DIR=tests/config/no-options');
```

Comment:
I'll change to `SimpleSAML_Configuration::loadFromArray(array(), '[ARRAY]', 'simplesaml');`